Tests/ArgumentParserUnitTests/HelpGenerationTests.swift

```diff
@@ -229,7 +228,7 @@ extension HelpGenerationTests {
           --lucky <numbers>       Your lucky numbers. (default: 7, 14)
           --optional/--required   Vegan diet. (default: --optional)
           --degree <degree>       Your degree.
-          --directory <directory> Directory. (default: current directory)
+          --directory <directory> Directory. (default: file:///path/to/file)
```

Comment:
hmm this should ideally use `url.path`

Reply:
Fixed, though this is all just internal to this test.